CONTRIBUTING.md

```diff
@@ -169,9 +171,9 @@ development environment visit your local [mailcatcher server](http://localhost:1
 ### Code Review
 
 * A Pull Request (PR) is created on GitHub. All PRs should have [a good commit
-  message](https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message).
-[The PR is attached to the related Trello
-card](https://blog.trello.com/github-and-trello-integrate-your-commits).
+  message](https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message). Optionally, include a 
+  link to the Tracker story in the PR summary. When a PR completes a Tracker story, mark that story as 'Finished' 
```

Comment:
should we mention the magical [Fixes #1234] thing here or nah?

Reply:
also, I wonder if we should make this a requirement rather than optionally. Much easier to code review when you can see the source ticket!